### DISCUSSION TITLE
Why are Vue Devtools still available for production build

### DISCUSSION THREAD

**Discussion by stone-zeng** [antfu/vite-ssg#329] [Q&A]
When using `vite-ssg build` to create a website, the Vue Devtools are still available. Is this intentional or do we have any options to control over this?

**Comment by stone-zeng** [comment]
I have noticed that by setting `NODE_ENV=production` we can make vite-ssg run in a "real" production mode. Is it possible to do this without explicitly setting the environment variable?

**Reply by xrayian** [reply]
I set it like that with a .env file still isn't working in my case

**Reply by Mini-Sylar** [reply]
@xrayian Did you find the solution to this?

**Reply by Mini-Sylar** [reply]
I found a solution @xrayian 

by explictly setting my node env in my biuld command it worked and disabled dev tools as well

for windows server
```
 "build": "set NODE_ENV=production&& vite-ssg build",
 ```
 
for linux server
```
  "build": "export NODE_ENV=production && vite-ssg build",
 ```

cheers

**Comment by sombriks** [comment]
maybe vite-ssg isn't passing .env files content to vite itself

**Reply by Mini-Sylar** [reply]
Hm interesting... I have some API keys in an .env file that are read normally though

I think someway somehow vite-ssg is unable to read NODE_ENV which is really odd.

I see these lines here. [Build.ts](https://github.com/antfu/vite-ssg/blob/ad77796edcec1946156c8ed4131c946631f97e47/src/node/build.ts#L29-L31). You would assume everytime ```build``` is run it should have set mode to production... I guess not

I also just tried setting vite mode to production and SSGOptions to production. It still run in dev mode.
